templates/openstack/support.html

```diff
@@ -129,40 +129,17 @@ <h2>For mission-critical infrastructure</h2>
   </div>
 </section>
 
-<section class="p-strip">
-  <div class="row">
-    <div class="col-8">
-      <h2>OpenStack support at the best server/annum cost</h2>
-
-      <p>Canonical&#39;s OpenStack support under <a href="/pro">Ubuntu Pro</a> subscriptions ensures the best cost efficiency. In 2021, KAUST chose Canonical as a vendor because of the best server/annum cost. Furthermore, with Canonical handling implementation and operational issues, the university has seen quicker problem resolution and significant savings from a manpower perspective. </p>
-
-      <a href="/engage/openstack-cloud-kaust">Read the case study&nbsp;&rsaquo;</a>
-    </div>
-
-    <div class="col-4 u-hide--medium u-hide--small u-vertically-center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/0853cdb4-kaust.jpeg",
-        alt="",
-        width="200",
-        height="200",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
 
 <section class="p-strip--light">
```

Comment:
```suggestion
<section class="p-strip">
```
Might be good to update this strip to add some division between these sections, as right now they look a bit weird with the same background color